source/common/router/rds_impl.cc

```diff
@@ -199,6 +199,7 @@ void RouteConfigProviderManagerImpl::addRouteInfo(const RdsRouteConfigProvider&
                                                   Buffer::Instance& response) {
   // TODO(junr03): change this to proto with JSON transcoding when #1522 is done.
   response.add("{\n");
+  response.add(fmt::format("    \"version_info\": \"{}\",\n", provider.versionInfo()));
   response.add(fmt::format("    \"route_config_name\": \"{}\",\n", provider.routeConfigName()));
   response.add(fmt::format("    \"cluster_name\": \"{}\",\n", provider.clusterName()));
   response.add("    \"route_table_dump\": ");
```

Comment:
while you here can you add a `\n` https://github.com/lyft/envoy/blob/8f12925cb4d4c85b55d05954892f565c883f9ca5/source/common/router/rds_impl.cc#L207